.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of changes, motivation and context for this PR.
+
+Fixes # (issue)
+
+## Type of change
+
+Please tick all options which are relevant.
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Requires documentation updates
+
+# Testing
+
+Please describe the test(s) that you added and ran to verify your changes.
+
+## Test(s) description
+
+**Test Configuration**:
+
+* OS:
+
+# Checklist:
+
+- [ ] I have added tests that prove my fix is effective and/or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] I have used understandable variable names
+- [ ] I have squashed my commits into a single commit ready for review
+- [ ] I have run `git-clang-format` on my changes
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] My changes generate no new warnings
```

Comment:
Nice, but do you expect people to follow it :) ?

Reply:
Not sure whether people will follow it, but its good to have as a reminder to those putting in PRs.

Reply:
Yeah I have a similar take to @vgvassilev... maybe we can relax it to provide a set of guidelines rather than a strict template that we expect people to follow. I was wondering if something like this would be good : https://github.com/root-project/root/blob/master/CONTRIBUTING.md.  

I propose something like this should be sufficient:

[ci/Interpreter/docs...] PR TITLE

Description/Motivation
Fixes #issue 
* [ ]  tested changes locally
* [ ]  updated the docs (if necessary)

what do you think @vgvassilev @mcbarton @alexander-penev